keyboards/helix/rev3_5rows/info.json

```diff
@@ -0,0 +1,19 @@
+{
+  "keyboard_name": "Helix rev3 5rows",
+  "url": "",
+  "maintainer": "yushakobo",
+  "width": 15,
+  "height": 5,
+  "layouts": {
+    "LAYOUT": {
+      "key_count": 64,
```

Comment:
```suggestion
```

Reply:
I have a question.

Why remove `key_count` from `helix/rev3_5rows/info.json` and not remove `key_count` from `helix/rev3_4rows/info.json`?